src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie,
+};
+
+export const MovieCard: React.FC<Props> = (props: Props) => {
+  const { movie } = props;
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
  const { prop1, prop2, prop3 } = movie;
```
But you should change `prop` for the proper names, that's just an example